agenda-webapps/src/main/webapp/vue-app/agenda-common/components/event/view/AgendaEventAttendeeItem.vue

```diff
@@ -33,6 +33,18 @@
       :class="responseIconResponse"
       class="my-auto text-right"></span>
   </div>
+  <div v-else-if="isAttendeeGuest" class="attendee">
+    <v-avatar
+      left
+      size="32"
+      class="mx-0 my-auto ">
+      <v-img
+        src="/portal/rest/v1/social/users/default-image/avatar" />
+    </v-avatar>
+    <span class="mx-1 my-auto agenda-guest">
```

Comment:
Please refer to the list of Helpers class here https://github.com/Meeds-io/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/helpers.less. In fact, in general we avoid defining classes for colors, margins, spacing, flex box ... in order to make the project :
1/ Brandable
2/ RTL support
3/ Maintainable
4/ Better Performances when reducing the defined CSS classes

IMO, using the class `primary--text` will allow you to access the primary color property.

```suggestion
    <span class="mx-1 my-auto primary--text">
```

Could you please delete the class definition of `agenda-guest`.

Reply:
Done.